website/project/views/contributor.py

```diff
@@ -378,6 +378,55 @@ def project_manage_contributors(auth, node, **kwargs):
     return {}
 
 
+@no_auto_transaction
+@must_be_valid_project  # injects project
+@must_have_permission(ADMIN)
+@must_not_be_registration
+def project_remove_contributor(auth, **kwargs):
+    """Remove a contributor from a list of nodes.
+
+    :param Auth auth: Consolidated authorization
+    :param-json list contributors: Ordered list of contributors represented as
+        dictionaries of the form:
+        {'id': <id>, 'permission': <One of 'read', 'write', 'admin'>}
```

Comment:
Let's review this whole docstring. Where does it show up? Where are our docs published?